main.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from matplotlib import pyplot as plt
+from quickMafs import quickMafs
+
+def main():
+
+    np.set_printoptions(precision = 3, suppress = True) #3 sigfigs, no sci notation

```

Comment:
yo, personally comments at the end of lines are the most annoying and useless way to comment. Idk what standard is in python, but I prefer them on the line preceding whats being discussed.  Its a pain in the ass to have to scroll across the text editor to read comments

#comments like this are easier to see and read
#comments like this make the user read what
#your intent was before you do it
np.myExample()